TFT/src/User/Configuration.h

```diff
@@ -62,12 +66,12 @@
  * Options: BLACK, BLUE, BROWN, BRRED, CYAN, GBLUE, GRAY, GREEN, MAGENTA, RED, WHITE, YELLOW
  */
 #define ST7920_BKCOLOR BLACK
-#define ST7920_FNCOLOR YELLOW
+#define ST7920_FNCOLOR CYAN
 
 /**
  *  Text displayed at the top of the TFT in Marlin Mode.
  */
-//#define ST7920_BANNER_TEXT "LCD12864 Simulator"
+//#define ST7920_BANNER_TEXT "XY-2 PRO"
```

Comment:
please remove settings specific to your printer and try to use default values in PRs.

Reply:
ok, i will do it in the future. thank you for your help.

Reply:
Please update this PR to to remove your specific printer settings. It’ll get merged otherwise unless BigTreeTech reverts the change.

Reply:
done..